samples/pushBatchOfDocumentsFromFolder.ts

```diff
@@ -33,12 +33,10 @@ async function main() {
     '/path/to/fileB.json',
   ];
 
-  const {onBatchUpload, onBatchError, done} =
-    await source.batchUpdateDocumentsFromFiles('my_source_id', entries);
-
-  onBatchUpload(onSuccessCallback);
-  onBatchError(onErrorCallback);
-  await done();
+  await source
```

Comment:
You are missing an await.
`batchUpdateDocumentsFromFiles` returns a promise, so you cannot really use the builder pattern here.
![image](https://user-images.githubusercontent.com/12199712/159358241-260f82c9-193c-435c-93b6-30d7b759e986.png)